x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/expandable_section/expandable_section_results.tsx

```diff
@@ -227,13 +232,56 @@ export const ExpandableSectionResults: FC<ExpandableSectionResultsProps> = ({
   );
 
   const openCustomUrl = (item: DataGridItem, customUrl: MlKibanaUrlConfig) => {
-    // Replace any tokens in the configured url_value with values from the source record and open link in a new tab/window.
-    const urlPath = replaceTokensInDFAUrlValue(
-      customUrl,
-      item,
-      data.query.timefilter.timefilter.getTime()
-    );
-    openCustomUrlWindow(urlPath, customUrl, basePath.get());
+    // If url_value contains $earliest$ and $latest$ tokens, add in times to the source record.
+    // Create a copy of the record as we are adding properties into it.
+    const record = cloneDeep(item);
+    const timestamp = record[indexPattern!.timeFieldName!];
+    const configuredUrlValue = customUrl.url_value;
+    const timeRangeInterval =
+      customUrl.time_range !== undefined ? parseInterval(customUrl.time_range) : null;
+    const intervalString = timeRangeInterval?.humanize();
+    const interval = supportedUnits.find((unit) => intervalString?.includes(unit)) ?? 'day';
```

Comment:
NOTE: Was going to update this to use 'day' unless the interval is >= 'hour' since that would seem to make sense when going back a month, year, etc to start at the start of the day of the start of the month, year, etc. If that makes sense.

Reply:
The time range behavior on drilldown LGTM with the changes in https://github.com/elastic/kibana/pull/159874/commits/124d047b2f4b880c65e2cec0bbd980421b74b4bd.

Reply:
Simplified behavior as discussed in 124d047b2f4b880c65e2cec0bbd980421b74b4bd